src/components/PopupWithForm.js

```diff
@@ -1,4 +1,7 @@
 import Popup from "../components/Popup.js";
```

Comment:
Please, delete ` const currentModal = document.querySelector(".modal_opened");` from line 16 in `Popup`. You don't use it and you can't use it